kong/client.go

```diff
@@ -164,6 +167,28 @@ func NewClient(baseURL *string, client *http.Client) (*Client, error) {
 	return kong, nil
 }
 
+// SetWorkspace defines a workspace for the client
+func (c *Client) SetWorkspace(workspace string) {
+	c.workspaceLock.Lock()
+	defer c.workspaceLock.Unlock()
+	c.workspace = workspace
+}
+
+//GetWorkspace return the workspace
+func (c *Client) GetWorkspace() string {
```

Comment:
Idiomatic Go name for this function is `Workspace()`. Please update.
Let's wait for @mflendrich to respond before making the code change to avoid conflicting comments from me and Michal at the same time.

Reply:
It seems like it (https://golang.org/doc/effective_go#Getters), indeed. I'll wait.

Reply:
Sounds good to me too :+1: